Source/WebCore/html/HTMLStyleElement.cpp

```diff
@@ -101,12 +101,28 @@ void HTMLStyleElement::attributeChanged(const QualifiedName& name, const AtomStr
         if (auto* scope = m_styleSheetOwner.styleScope())
             scope->didChangeStyleSheetContents();
         break;
+    case AttributeNames::blockingAttr:
+        if (m_blockingList)
+            m_blockingList->associatedAttributeValueChanged();
+        break;
     default:
         HTMLElement::attributeChanged(name, oldValue, newValue, attributeModificationReason);
         break;
     }
 }
 
+DOMTokenList& HTMLStyleElement::blocking()
```

Comment:
Link to: https://html.spec.whatwg.org/multipage/semantics.html#dom-style-blocking

Reply:
Also is there some unification to be done between how style blocks rendering and everything else blocks rendering?